pkg/cloudclient/gcp/private.go

```diff
@@ -210,13 +214,13 @@ func (c *Client) findGCPForwardingRuleForExtIP(rhapiLbIP string) (*compute.Forwa
 	var fr *compute.ForwardingRule
 	for _, lb := range response.Items {
 		if lb.IPAddress == rhapiLbIP {
-			fr = lb
+			return fr, nil
 		}
 	}
 	if fr == nil {
 		return nil, fmt.Errorf("Forwarding rule not found in GCP for given service IP %s", rhapiLbIP)
 	}
-	return fr, nil
+	return nil, nil
```

Comment:
This should still return an error "LB not found"

Reply:
gotcha